### DISCUSSION TITLE
Gerätefunktionen 2-4 getrennt nach Front/Heck bedienen

### DISCUSSION THREAD

**Discussion by flattergaul** [Mogli12/KeyboardSteer#754] [Ideas]
Hallo Stefan,

erst mal vielen, vielen Dank für VCA. Es erleichtert das LS-Leben und macht es realer.

Wäre es auch möglich, die Steuerung der weiteren Gerätefunktionen 2-4 auch getrennt nach Front/Heck einzubauen?
So wie es bei Heben/Senken bzw. Aktivieren/Deaktivieren ist. Dann würden wir uns das Umschalten zwischen den Geräten z.B. um Front- und Heckmähwerk aufzuklappen, ersparen.

Das würde der realen Möglichkeit bei modernen Traktoren Zusatzfunktionen von Geräten vorhandenen Bedienelementen auf der Multifunktionsarmlehen zuzuordnen sehr nahe kommen.

Liebe Grüße
Michael

**Comment by Unix0815** [comment]
Gibt's hierzu irgendwas neues? Es wäre wirklich super, wenn man Geräte wie zB Mähwerk vorne und hinten getrennt auf und zuklappen (Taste X) könnte ohne mühsam durch sämtliche Geräte wechseln zu müssen.

**Comment by Mogli12** [comment]
Hallo Michael,
das ist recht schwierig. Dieselbe Taste wird von verschiedenen Spezialisierungen unterschiedlich verwendet. Es ist leider nicht so, dass man für ein Gerät einfach die Standardfunktion für Taste X aufrufen kann.
Viele Grüße,
Stefan
